### PR TITLE
Refactoring of geolocator and put config into database

### PR DESCRIPTION
**Pull request by ddegasperi**

### CODE REVIEW

File: doc/update_db_from_3.4.0.sql [hunk 1]
```diff
@@ -1594,4 +1594,12 @@ SELECT max(version_name) INTO v_author_version FROM version where version_key =
         v_author_version = '3.6.1';
         INSERT INTO version (version_name,version_key, version_date) values ('3.6.1', 'author', '2019-05-31');
     END IF;
+    RAISE INFO v_author_version ;
+    IF v_author_version = '3.6.1' THEN
+        ALTER TABLE mapset ADD COLUMN geolocator jsonb ;
```
Comment (ssegala) [inline doc/update_db_from_3.4.0.sql]: Shall we add IF NOT EXISTS?

**Comment by fontsie** [comment]
Maybe is needed a symfony command to migrate from the config to the database?
In some particular case geolocator configuration is store in custom directory `$GEOLOCATOR_CONFIG_PATH` as json file. This file is managed by custumer. How they can update config in db now? Maybe with a watcher on file system?

**Comment by ddegasperi** [comment]
> Maybe is needed a symfony command to migrate from the config to the database?
> In some particular case geolocator configuration is store in custom directory `$GEOLOCATOR_CONFIG_PATH` as json file. This file is managed by custumer. How they can update config in db now? Maybe with a watcher on file system?

@fontsie I've updated the PR and add the geolocator field accessible from the UI. This modification allows the customer to update their geolocator configuration directly from the author.

**Review by fontsie — APPROVED** [review]
Per me ok

**Pull request merged**